app/src/main/java/com/name/name/musicappmvp/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,160 @@
+                    if (playSongService?.isPause == true) playSongService?.start()
+                    playSongService?.playing = true
+                    buttonPlay.setImageResource(R.drawable.ic_pause)
+                } else {
+                    playSongService?.pause()
+                    playSongService?.playing = false
+                    buttonPlay.setImageResource(R.drawable.ic_play)
+                }
+            }
+            R.id.buttonNext -> {
+                PlaySongService.position++
+                if (PlaySongService.position == localSongs.size) PlaySongService.position = 0
+                resetPlayingState()
+            }
+            R.id.buttonPrev -> {
+                PlaySongService.position--
+                if (PlaySongService.position < 0) PlaySongService.position = localSongs.size - 1
+                resetPlayingState()
+            }
+        }
+    }
+
+    private fun checkPermission() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            if ((checkSelfPermission(Manifest.permission.READ_EXTERNAL_STORAGE) != PackageManager.PERMISSION_GRANTED)) {
+                requestPermissions(arrayOf(Manifest.permission.READ_EXTERNAL_STORAGE), 1)
+                return
+            }
+        }
+    }
+
+    override fun displayLocalSong() {
+        localSongs = localMusicPresenter?.getLocalSong(this) as MutableList<LocalSong>
+        localSongs.sortWith(compareBy({ it.title }, { it.title }))
+        val adapter = LocalMusicAdapter(::showInformationSong)
+        adapter.sendList(localSongs)
+        recyclerSongs.adapter = adapter
+        handleButton()
+    }
+
+    override fun handleButton() {
+        buttonPlay.setOnClickListener(this)
+        buttonNext.setOnClickListener(this)
+        buttonPrev.setOnClickListener(this)
+        songDuration.setOnSeekBarChangeListener(object : OnSeekBarChangeListener{
+            override fun onProgressChanged(seekBar: SeekBar?, progress: Int, fromUser: Boolean) {
+                playSongService?.rewind(progress)
+                duration.text = playSongService?.timeTracking()
+                if (playSongService?.isNext == true){
+                    textNameSong.text = getString(R.string.text_playing).plus(" " +localSongs[PlaySongService.position].title)
+                }
+            }
+
+            override fun onStartTrackingTouch(seekBar: SeekBar?) {
+            }
+
+            override fun onStopTrackingTouch(seekBar: SeekBar?) {
+            }
+
+        })
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        unbindService(connection)
+        isBind = false
+    }
+
+    private fun resetPlayingState() {
+        songDuration.progress = 0
+        playSongService?.playSong()
+        songDuration.max = localSongs[PlaySongService.position].duration
+        textNameSong.text = getString(R.string.text_playing).plus(" " +localSongs[PlaySongService.position].title)
```

Comment:
Đoạn này anh nên khai báo 1 string trong strings.xml dưới dạng format : playing %s
```suggestion
        textNameSong.text = getString(R.string.format_name_song, name)
```